contexts/GetEXAContext.tsx

```diff
@@ -399,15 +426,18 @@ export const GetEXAProvider: FC<PropsWithChildren> = ({ children }) => {
       const { status, transactionHash } = await waitForTransaction({ hash: txHash_ });
       setTX({ status: status ? 'success' : 'error', hash: transactionHash });
       setScreen(Screen.TX_STATUS);
+      transaction.purchase('getEXA', input);
```

Comment:
check the tx status before submitting a `purchase`

Reply:
tks